samples/requirement-targeting-oneway-dependency/nodejs/README.md

```diff
@@ -0,0 +1,227 @@
+1. Select **File > Open Folder** in VS Code and choose this samples directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+## Run the app (Manually Uploading to Teams)
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+### 1. Setup for Bot
+- In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2)
+
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+
+- While registering the bot, use `https://<your_tunnel_domain>/api/messages` as the messaging endpoint.
+    > NOTE: When you create your bot you will create an App ID and App password - make sure you keep these for later.
+
+### 2. Setup NGROK
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http 3978 --host-header="localhost:3978"
+    ```
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3978 --allow-anonymous
+   ```
+
+### 3. Setup for code
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+2) In a terminal, navigate to `samples/requirement-targeting-oneway-dependency`
+
+3) Install modules
+
+    ```bash
+    npm install
+    ```
+
+4) Update the `custom-environment-variables` configuration for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword`, `BaseUrl` with application base url.
+
+5) Update the `default` configuration for the bot to use the `appId` and `appPassword`.
+
+5) Run your app
+
+    ```bash
+    npm start
+    ```
+
+## RT Sample App Feature
+**This sample app feature is to show oneway dependency**
+
+| Manifests | Capabilities | Dependency |
+|----------|----------|----------|
+| 1.OneWay ME-DependsOn-Bot | Bot and Action-ME | Action-ME Depends on Bot |
+| 2.OneWay Tab-DependsOn-Bot | Static-Tab and Bot | Static-Tab Depends on Bot |
+| 3.StaticTab-With-DialogURL |Static-Tab with dialog URL | Host Support Functionalities |
+
+1) actionME depends on Bot => for the bot-base-actionME scenario 
```

Comment:
This is not very clear. All actionMEs are bot-based. This scenario is specifically for actionME that contains bot-sent-card scenario. So it should probably be "to specify the actionME contains a bot-sent-card scenario and has to depend on bot to work properly".

Reply:
fixed